ci-operator/step-registry/baremetalds/e2e/test/baremetalds-e2e-test-commands.sh

```diff
@@ -34,8 +34,18 @@ scp "${SSHOPTS[@]}" /usr/bin/openshift-tests /usr/bin/kubectl "root@${IP}:/usr/l
 
 # Tests execution
 set +e
-echo "### Running tests"
-ssh "${SSHOPTS[@]}" "root@${IP}" openshift-tests run "openshift/conformance/parallel" --dry-run \| grep 'Feature:ProjectAPI' \| openshift-tests run -o /tmp/artifacts/e2e.log --junit-dir /tmp/artifacts/junit -f -
+if [[ -s "${SHARED_DIR}/test-list" ]]; then
+    echo "### Copying test-list file"
+    scp "${SSHOPTS[@]}" "${SHARED_DIR}/test-list" "root@${IP}:/tmp/test-list"
+    echo "### Checking test-list file"
+    ssh "${SSHOPTS[@]}" "root@${IP}" ls -l /tmp/test-list 
+    ssh "${SSHOPTS[@]}" "root@${IP}" cat /tmp/test-list 
+    echo "### Running tests"
```

Comment:
Probably the above two ssh commands are not required

Reply:
Removed leftover debugs.